src/Symfony/Component/VarDumper/Caster/OpenSslCaster.php

```diff
@@ -0,0 +1,92 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\VarDumper\Caster;
+
+use Symfony\Component\VarDumper\Cloner\Stub;
+
+/**
+ * @author Alexandre Daubois <alex.daubois@gmail.com>
```

Comment:
hum, many `@author` mentions are partial or not accurate...

Reply:
Indeed, copy pasted without thinking enough of the annotation. Fixing it in the next commit 👍